ui/media/js/main.js

```diff
@@ -683,6 +683,12 @@ async function checkTasks() {
 
     const genSeeds = Boolean(typeof task.reqBody.seed !== 'number' || (task.reqBody.seed === task.seed && task.numOutputsTotal > 1))
     const startSeed = task.reqBody.seed || task.seed
+
+	// update the seed *before* starting the processing, so interrupting the processing retains the latest seed
+    if (randomSeedField.checked) {
```

Comment:
Sounds okay. Can you please remove the previous code (now a duplicate) from line 740?

Reply:
Done.